### ISSUE TITLE
What is the average salary?

### ISSUE THREAD

**Issue by Kl0tzfaust** [edenspiekermann/ama#1]
What is the average salary of a normal Graphic Designer at edenspiekermann?

**Comment by harryfk** [comment]
* gets popcorn

**Comment by sdietrichberlin** [comment]
Excellent question! But we only have excellent graphic designers ;-)

In all seriousness - salary depends on the level of experience, skill set, the market, and other factors. Not sure an average number would be very significant - sorry.

**Comment by HugoGiraudel** [comment]
Hey! Thanks for asking. Allow me to enhance Sebastian's answer with some extra information that one might be interested in knowing.

I believe salary is not the only thing that one should pay attention to. For instance, every employee (including myself) at Edenspiekermann gets 30 days a year of paid vacation as well as personal budget for educational purposes. Like, anything. Not necessarily related to the tech industry. I could take piano lessons, or cooking training (not that I would need it though :D), it’s all about learning something new.

On top of that, we have what we call a *brainfood* every week, where someone (internal or external from the company) comes and speaks about some topic, which might or might not be related to work. Then, we all have a breakfast together. Hence the *brainfood* name. First we brain, then we food. Speaking of food, there is always drinks and snacks here, as well as icecream when everything is melting outside. Along the same lines as brainfood, developers and designers gather once a week to exchange and brainstorm about their job and how we can improve things together.

We also organize 2 to 3 events a year called *[Maker Days](http://www.edenspiekermann.com/blog/espi-maker-days-2)* during which we group in teams to build fun projects, not necessarily related to the company at all. It lasts 2 week days, and ends with a great and friendly evening together where we can enjoy ourselves. Although I haven’t been long enough at ESPI, I’ve heard good things about the summer and new years parties as well. ;)

Long story short, it is hard to tell what is the average salary of a designer (or basically any role) in Edenspiekermann, especially since we have a lot of different levels: interns, juniors, seniors, partners... That being said, on top of a decent salary, Edenspiekermann values its employees and tries to make everyone feel at home. 

Last but not least, and I suppose that’s rather personal but I feel like it’s very nice to be able to work with creative and talented minds on a daily basis. :)
